pywbem_mock/_instancewriteprovider.py

```diff
@@ -277,114 +220,117 @@ def __init__(self, cimrepository=None):
 
     ####################################################################
     #
-    #   CreateInstance server request
+    #   CreateInstance responder
     #
     ####################################################################
 
     def CreateInstance(self, namespace, NewInstance):
         """
-        Implements a WBEM server responder for
+        Implements a mock WBEM server responder for
         :meth:`pywbem.WBEMConnection.CreateInstance`.
 
-        Create a CIM instance in the local CIM repository of this class.
+        Create a new CIM instance in the CIM repository of the mock WBEM server.
 
-        The input parameters must have been validated before this method
-        is called including:
+        Validation that already has been performed by the caller:
```

Comment:
drop "that has".  Describe where this is in code.

Reply:
It is not in this code, which is why it states that it has been done by the caller.